patches/minecraft/net/minecraft/client/renderer/EntityRenderer.java.patch

```diff
@@ -17,7 +17,40 @@
          float f4 = (float)this.field_78531_r.field_71474_y.field_151451_c / 32.0F;
          float f2 = f3 * (1.0F - f4) + f4;
          this.field_78539_ae += (f2 - this.field_78539_ae) * 0.1F;
-@@ -412,7 +413,7 @@
+@@ -355,23 +356,9 @@
+                 double d0 = (double)this.field_78531_r.field_71442_b.func_78757_d();
+                 this.field_78531_r.field_71476_x = entity.func_174822_a(d0, p_78473_1_);
+                 Vec3d vec3d = entity.func_174824_e(p_78473_1_);
+-                boolean flag = false;
+                 int i = 3;
+                 double d1 = d0;
+ 
+-                if (this.field_78531_r.field_71442_b.func_78749_i())
+-                {
+-                    d1 = 6.0D;
+-                    d0 = d1;
+-                }
+-                else
+-                {
+-                    if (d0 > 3.0D)
+-                    {
+-                        flag = true;
+-                    }
+-                }
+-
```

Comment:
this patch takes too many lines, can be cleaned up:
* no need to delete `flag`, it's not used below anymore so removing it just adds lines to the patch. The optimizer will get rid of it at runtime.
* no need to patch out the entire thing, just make the block impossible to run by adding an if false.

E.g. 
```Java
if (false)
if (this.field_78531_r.field_71442_b.func_78749_i())
...

doing those two things will take this from a 13 line patch to a 1 line one

Reply:
Hmm, those are good suggestions. Thanks.